rental.rb

```diff
@@ -8,4 +9,22 @@ def initialize(date, person, book)
 
     @book = book
   end
+
+  def person=(person)
+    @person = person
+    @person.rentals.push(self) unless @person.rentals.include?(self)
+  end
+
+  def book=(book)
+    @book = book
+    @book.rentals.push(self) unless @book.rentals.include?(self)
+  end
+
+  def to_hash
+    hash = {}
+    hash[:date] = @date
+    hash[:person_id] = @person.id
+    hash[:book_id] = @book.id
+    hash
```

Comment:
- [ ] Kindly fix the error that appears when the user exits the program.
![image](https://user-images.githubusercontent.com/62290545/181072698-e84d1360-c4f7-4aac-81f2-0857cfc986d7.png)